content/blog/using-graphql-with-the-filesystem.md

```diff
@@ -391,4 +385,4 @@ query BlogPostQuery($relativePath: String!) {
 
 ## Conclusion
 
-TODO
+TODO
```

Comment:
- Sum up the benefits of having type checking on top the filesystem
- Added benefits: better developer experience in Visual Code?
-  invite people to play with the starter and give us feedback